src/privatesend/privatesend-server.cpp

```diff
@@ -845,7 +845,7 @@ void CPrivateSendServer::PushStatus(CNode* pnode, PoolStatusUpdate nStatusUpdate
 {
     if (!pnode) return;
     CNetMsgMaker msgMaker(pnode->GetSendVersion());
-    connman.PushMessage(pnode, msgMaker.Make(NetMsgType::DSSTATUSUPDATE, nSessionID, (int)nState, (int)vecEntries.size(), (int)nStatusUpdate, (int)nMessageID));
+    connman.PushMessage(pnode, msgMaker.Make(NetMsgType::DSSTATUSUPDATE, nSessionID, (int)nState, 0, (int)nStatusUpdate, (int)nMessageID));
```

Comment:
maybe put a TODO or something somewhere to remind ourselves we should remove this fully at some point and do a proto bump?

Reply:
This is an outdated change and was later replaced with CPrivateSendStatusUpdate

Reply:
:+1: